app/assets/scripts/components/map/location-layer.js

```diff
@@ -51,13 +57,12 @@ export default function LocationLayer({
         id: 'location-layer',
         source: sourceId,
         'source-layer': 'default',
-        filter: ['in', 'locationId', ['literal', locationIds]],
+        filter: ['in', ['get', 'locationId'], ['literal', locationIds]],
         type: 'circle',
         paint: {
-          'circle-color': {
-            property: 'lastValue',
-            stops: generateColorStops(activeParameter),
-          },
+          'circle-color': isCore(activeParameter)
+            ? getFillExpression(activeParameter)
```

Comment:
I'm seeing a strange behavior here. When this is a core parameter and a reference grade sensor, I see both square and circle. The example I'm looking at is `http://localhost:3000/#/location/62302` nearby locations with an active parameter of `PM2.5`

Reply:
good catch! I did not update the highlight location layer to use the new symbology